docs/_includes/markup/buttons-previous-next-page.njk

```diff
@@ -1,14 +1,14 @@
-<div class="btns-previous-forward">
+<div class="btns-previous-next">
```

Comment:
Not sure why this does not have the css update with it to verify it appears correctly. Can you combine the updates or place this with the css branch?